tests/test_serde.rs

```diff
@@ -94,7 +94,7 @@ fn test_map() {
     let yaml = unindent(r#"
         ---
         x: 1
-        y: 2"#);
+        "y": 2"#);
```

Comment:
I think this is wrong and there is some bug... @dtolnay

Reply:
This is from https://github.com/chyh1990/yaml-rust/pull/72.

Reply:
Yes, this is probably an issue with the emitter. But this is more of a cosmetic issue, as it is still valid YAML with the same meaning.

Reply:
this is intended to keep any other yaml1.1 (not 1.2) parser from interpreting this as a [boolean](http://yaml.org/type/bool.html). I am currently unaware whether this ought to be treated differently when the value is a key.

Reply:
I have tried the yaml parser in ruby 2.3, 'Y', 'y', 'N', 'n' is not interpreted as boolean in both keys and values. But 'YES', 'yes', 'NO', 'no' is boolean in ruby. PyYAML also works like this.

I know it is violating YAML spec,  the using 'y' in key is a common case.

Reply:
the ruby parses an pyyaml are all 1.1 parsers